apps/examples/st_things/light/sample_light_user.c

```diff
@@ -34,6 +34,7 @@ static const char *PROPERTY_VALUE_COLOR_TEMPERATURE = "ct";
 
 static char *power_status[] = { "off", "on" };
 
+static char *power;
```

Comment:
Clould you please apply rule of variable naming. 
ex) g_

Reply:
fixed.